.github/workflows/ci.yml

```diff
@@ -51,7 +88,7 @@ jobs:
         env:
           TOXENV: ${{ matrix.PYTHON.TOXENV }}
 
-      - uses: codecov/codecov-action@v1
-        with:
-          name: "Python ${{ matrix.PYTHON.VERSION }} on ${{ matrix.WINDOWS.WINDOWS }}"
-          fail_ci_if_error: true
+      - name: Upload coverage
+        run: |
+          curl -o codecov.sh -f https://codecov.io/bash
+          bash codecov.sh -n "Python ${{ matrix.PYTHON.VERSION }} on ${{ matrix.WINDOWS.WINDOWS }}" -Z
```

Comment:
Looks like this carried over from the windows section

Reply:
Yes, I did it intentionaly. Consistency!